test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -0,0 +1,100 @@
+# Owner(s): ["module: onnx"]
```

Comment:
did you check whether the pipeline is calling this file?

Reply:
I think it is not, thanks for pointing out.  I don't want to touch this PR while it's being imported by Meta. I ran this test locally and confirms it passes, and after this is merged we can open another PR to run this automatically.

Reply:
Tests under `test/onnx` by default runs in shard 1.